tgui/packages/tgui-say/handlers/arrowKeys.tsx

```diff
@@ -1,14 +1,13 @@
-import { KEY_DOWN, KEY_UP } from 'common/keycodes';
 import { getHistoryLength } from '../helpers';
 import { Modal } from '../types';
 
 /** Increments the chat history counter, looping through entries */
-export const handleArrowKeys = function (this: Modal, direction: number) {
+export const handleArrowKeys = function (this: Modal, direction: string) {
 	const { historyCounter } = this.fields;
-	if (direction === KEY_UP && historyCounter < getHistoryLength()) {
+	if (direction === 'ArrowUp' && historyCounter < getHistoryLength()) {
 		this.fields.historyCounter++;
 		this.events.onViewHistory();
-	} else if (direction === KEY_DOWN && historyCounter > 0) {
+	} else if (direction === 'ArrowDown' && historyCounter > 0) {
```

Comment:
Why?

Reply:
keycode is deprecated and i'm tired of looking at the error
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
What error?